.travis.yml

```diff
@@ -37,11 +38,19 @@ jobs:
         - chmod +x operator-sdk
         - mv operator-sdk $GOPATH/bin/
 
+        # Download OPM tool
+        - curl -Lo opm https://github.com/operator-framework/operator-registry/releases/download/v${OPM_VERSION}/linux-amd64-opm
+        - chmod +x opm
+        - mv opm $GOPATH/bin/
+
       script:
         # Run merge-to-master-release
         - make merge-to-master-release
         - make push-to-manifest-repo
         - make prepare-bundle-to-quay
+        - make build-bundle-image
+        - make push-bundle-image
+        - make build-bundle-index-image
```

Comment:
since we are adding more Makefile rules, instead let's try to consolidate and have just one rule that does what we want, something like `push-bundle-index-image` - the rest of the targets should be dependencies to it.